webassets_elm/tests.py

```diff
@@ -7,14 +7,42 @@
 register_filter(Elm)
 
 
+ELMJSON = """
+{
+    "type": "package",
+    "name": "webassets-elm/test",
+    "summary": "this is only here because we want to be able to run tests",
+    "license": "BSD-3-Clause",
+    "version": "1.0.0",
+    "exposed-modules": [],
+    "elm-version": "0.19.0 <= v < 0.20.0",
+    "dependencies": {
+        "elm/core": "1.0.0 <= v < 2.0.0",
+        "elm/html": "1.0.0 <= v < 2.0.0"
+    },
+    "test-dependencies": {}
+}
+"""
```

Comment:
This data is only needed by the Elm compiler *during testing*, as is the input file. There is no reason to have this kick around as a separate file in the project, and actually serves to confuse if it does live on the filesystem (*there's an `elm.json` file, but no Elm source files?*). So I strongly prefer to keep this as close to the tests as possible.

In the same vein, we don't need any access to the contents from Python here, this is an opaque blob only needed to make the integration with the compiler work. And since the `TempEnvironmentHelper` base class expects strings in the `default_files` mapping, if you do store this as a Python dictionary you'd have to serialise to JSON first.

So I don't see any advantages to changing this back to a file on disk, nor to storing it as a Python dict.